### ISSUE TITLE
bandwith unit should be optionally bps

### ISSUE THREAD

**Issue by alfrenovsky** [tgraf/bmon#9]
bmon measures in Bytes per second. Wich is a useless in some scenarios.

I should be at least an option to change the units to bits per second.

**Comment by notgood** [comment]
At the moment bmon uses all-or-nothing approach to units. Every field is either MiB, or MB, or Mbit.

But most of the times, when it comes to the network "speed", is customary to use bits. And in order to display "amount" transferred, base2 MiB or base10 MB are usually used. (religion thing, really)

It would be nice to allow to configure unit scale per field, or at least allow to customize it separately for "speed" and "amount" fields.

P.S. bmon is hands down the best console console bandwidth I've encountered so far, using it on numerous embedded openwrt routers :) Good job!

**Comment by tgraf** [comment]
I agree with this and would like to accept a pull request for this feature.

**Comment by ogarod** [comment]
Is the -b option meant to apply to output modules as well? Here's how I am using the `format` output module:

```
-o format:fmt='$(element:name)\t$(attr:rx:bytes)\t$(attr:rxrate:bytes)\t$(attr:rx:packets)\t$(attr:rxrate:packets)\n'
```

However it seems the units used are still bytes, even when I pass in the -b flag.

**Comment by twisted-nematic57** [comment]
Seems like the units are all bits in bmon 4.0 when you pass `--use-bit`, but the graph title text still says bytes, even though it's actually displaying bits.

**Comment by adavidzh** [comment]
> Seems like the units are all bits in bmon 4.0 when you pass `--use-bit`, but the graph title text still says bytes, even though it's actually displaying bits.

I just noticed exactly this.
